.circleci/config.yml

```diff
@@ -319,16 +328,15 @@ jobs:
       - run:
           name: Authorize gcloud
           command: |
-            if [[ "$CIRCLE_PROJECT_USERNAME" == "getodk" ]]; then \
+            if [[ -n "$GCLOUD_SERVICE_KEY" ]]; then \
               gcloud config set project api-project-322300403941
               echo $GCLOUD_SERVICE_KEY | base64 --decode > client-secret.json
               gcloud auth activate-service-account --key-file client-secret.json
             fi
       - run:
           name: Run integration tests
           command: |
-            if [[ "$CIRCLE_PROJECT_USERNAME" == "getodk" ]]; then \
-              echo "y" | gcloud beta firebase test android run \
+            echo "y" | gcloud beta firebase test android run \
```

Comment:
Should this also be wrapped in the check for the service key?

Reply:
Actually, what is the advantage of checking? Should these not just fail if `GCLOUD_SERVICE_KEY` isn't set? Otherwise, you'd end up with this job passing having done nothing. That would be bad in our situation, but also for a fork that might not realize they're not actually running integration tests.